bin/p2-watch/main.go

```diff
@@ -22,6 +22,7 @@ var (
 	consulToken   = kingpin.Flag("token", "The ACL to use for accessing consul.").String()
 	headers       = kingpin.Flag("header", "An HTTP header to add to requests, in KEY=VALUE form. Can be specified multiple times.").StringMap()
 	https         = kingpin.Flag("https", "Use HTTPS").Bool()
+	waitTime      = kingpin.Flag("wait", "Duration to wait for Consul changes. Useful for proxy services that time out on long-polling requests.").Default("30s").Duration()
```

Comment:
done, improved help text is: `Maximum duration for Consul watches, before resetting and starting again.`